common/istio-1-11/README.md

```diff
@@ -0,0 +1,93 @@
+# Istio
+
+## Upgrade Istio Manifests
+
+Istio ships with an installer called `istioctl`, which is a deployment /
```

Comment:
Could the installation be done only with kustomizations?

Reply:
yes, with the single command stated [here](https://github.com/kubeflow/manifests#install-with-a-single-command)